server/dbSeed/userseed.js

```diff
@@ -0,0 +1,91 @@
+let mongoose = require("mongoose");
+let db = require("../models");
+
+mongoose.connect(
+  "mongodb://localhost/5sherpas",
+  {
+    useNewUrlParser: true,
+    useFindAndModify: false,
+    useUnifiedTopology: true,
+  },
+  () => {
+    console.log("Mongoose connected");
+  }
+);
+
+let users = [
+  {
+    name: "User1",
+    email: "user1@gmail.com",
+    password: "USER1",
+    avatar: 1,
+    challengeCategories: [
```

Comment:
can we create users without seed challenge data? we shouldn't have placeholders in here because users will always be created with out. If we have challengeCategories they should be actual categories so we know the user settings page is pulling them in correctly.

also have you tested this? I am not sure we can seed a user and login because the bcrypt is looking for a hashed password.. It might be able to match unhashed ones too but i do not know